docs/_markbind/boilerplates/codeAndOutputCode.md

```diff
@@ -21,3 +25,30 @@
 {{ code | safe }}
 </box>
 </div>
+{% endmacro %}
+
+{% if horizontal != null %}
+<div style="overflow-x: auto">
```

Comment:
does this allow indentation? If so we can indent this code, which is getting more complex than previous

Reply:
It does but it would be misaligned with the vertical layout
With indent: 
![image](https://user-images.githubusercontent.com/77204346/223294114-eff0dd58-5b9b-4049-9223-a135e1a17209.png)

Should I make the change then?

Reply:
Indent both this and the div from codeBox() and outputBox()? 
That would result in misalignment between vertical and horizontal layout since the horizontal layout would have two indentations 

Additional indentation:
![image](https://user-images.githubusercontent.com/77204346/223294604-8f93cfc8-2742-4dec-8dc7-a0272e50d3fb.png)

Current: 
![image](https://user-images.githubusercontent.com/77204346/223294845-269873b0-8387-49a5-8fc6-0f6ba5766165.png)

Reply:
ok, this could be misleading as the code is also the thing that we try to display. My apologies!

Reply:
Oh okok I see thanks! It doesn't seem to allow indentation. The html tags are not interpretated if I indent it.

Reply:
Interesting. The macros seem to have the interpretation issue you suggested, but the table portion seems to work fine (which is good because that's the part that I would have hoped to see the code properly formatted). Have you tried something like this:
```
{% if horizontal != null %}
<div style="overflow-x: auto">
    <table style="width: 100%">
        <tbody>
            <tr>
                <td style="width: 50%">
                {{ codeBox() }}
                </td>
                <td style="width: 50%">
                {{ outputBox() }}
                </td>
            </tr>
        </tbody>
    </table>
</div>

{% else %}

{{ codeBox() }}

{{ outputBox() }}

{% endif %}
```

If it works let's update accordingly.

Reply:
Yeah I did! But it ends up like this 
![image](https://user-images.githubusercontent.com/77204346/223392989-7cde0912-ab5d-4d18-99ca-a6eae0b9845b.png)